test/menu.test.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable func-names */
+import { mochaAsync, openDrawer, expectElementToExist } from './utils';
+import { createApplication, closeApplication } from './application';
+import { DEFAULT_GLOBAL_TIMEOUT } from './constants';
+import { QUIT_MENU_ITEM_ID, HOME_MENU_ITEM_ID } from '../src/config/selectors';
+
+describe('Menu Scenarios', function () {
+  this.timeout(DEFAULT_GLOBAL_TIMEOUT);
+  let app = null;
+
+  beforeEach(
+    mochaAsync(async () => {
+      app = await createApplication();
+    }),
+  );
+
+  afterEach(function () {
+    return closeApplication(app);
+  });
+
+  it(
+    'Displays correct navigation',
```

Comment:
Try to keep `it` blocks starting with lower case, to differentiate from `describe` blocks. Not important, just a tip.